pkg/ansible/run.go

```diff
@@ -148,12 +148,25 @@ func Run(flags *aoflags.AnsibleOperatorFlags) error {
 	}
 	// Create Service object to expose the metrics port(s).
 	// TODO: probably should expose the port as an environment variable
-	_, err = metrics.CreateMetricsService(context.TODO(), cfg, servicePorts)
+	service, err := metrics.CreateMetricsService(context.TODO(), cfg, servicePorts)
 	if err != nil {
 		log.Error(err, "Exposing metrics port failed.")
 		return err
 	}
 
+	// CreateServiceMonitors will automatically create the prometheus-operator ServiceMonitor resources
+	// necessary to configure Prometheus to scrape metrics from this operator.
+	services := []*v1.Service{service}
+	_, err = metrics.CreateServiceMonitors(cfg, namespace, services)
+	if err != nil {
+		log.Info("Could not create ServiceMonitor object", "error", err.Error())
+		// If this operator is deployed to a cluster without the prometheus-operator running, it will return
+		// ErrServiceMonitorNotPresent, which can be used to safely skip ServiceMonitor creation.
+		if err == metrics.ErrServiceMonitorNotPresent {
+			log.Info("Install prometheus-operator in your cluster to create ServiceMonitor objects", "error", err.Error())
+		}
+	}
+
```

Comment:
If the MonitorServices has been not generated currently, then we need to add a test to ensure that all ServiceMonitor also were gen as expected. It can be done [here](https://github.com/operator-framework/operator-sdk/blob/master/hack/tests/e2e-ansible.sh#L63).

Reply:
It's not clear if the necessary CRDs are installed in the CI environment to expect for the servicemonitors to be created.

Reply:
I think would be better we add a test, however, I am ok with move forward without it. 
@fabianvf wdyt? It this PR shows ok for me.

Reply:
I'm happy to add it if the ServiceMonitor kind exists in the CI environment. I just don't believe they do.

Reply:
solved. It was added.